addons/account/wizard/account_move_send_wizard.py

```diff
@@ -166,52 +179,132 @@ def _compute_pdf_report_id(self):
+    @api.depends('template_id')
+    def _compute_lang(self):
+        for wizard in self:
+            wizard.lang = self._get_default_mail_lang(wizard.move_id, wizard.template_id) if wizard.template_id else get_lang(self.env).code
+
+    @api.depends('template_id', 'lang')
+    def _compute_mail_partners(self):
         for wizard in self:
-            wizard.mail_template_id = self._get_default_mail_template_id(wizard.move_id)
+            wizard.mail_partner_ids = None
+
+            if wizard.template_id:
+                wizard.mail_partner_ids = self._get_default_mail_partner_ids(wizard.move_id, wizard.template_id, wizard.lang)
 
-    @api.depends('mail_template_id')
-    def _compute_mail_lang(self):
+    # Overrides of mail.composer.mixin
+    @api.depends('template_id', 'lang')
+    def _compute_subject(self):
         for wizard in self:
-            wizard.mail_lang = self._get_default_mail_lang(wizard.move_id, wizard.mail_template_id) if wizard.mail_template_id else get_lang(self.env).code
+            wizard.subject = None
+
+            if wizard.template_id:
+                wizard.subject = self._get_default_mail_subject(wizard.move_id, wizard.template_id, wizard.lang)
 
-    @api.depends('mail_template_id', 'mail_lang')
-    def _compute_mail_subject_body_partners(self):
+    # Overrides of mail.composer.mixin
+    @api.depends('template_id', 'lang')
+    def _compute_body(self):
         for wizard in self:
-            if wizard.mail_template_id:
-                wizard.mail_subject = self._get_default_mail_subject(wizard.move_id, wizard.mail_template_id, wizard.mail_lang)
-                wizard.mail_body = self._get_default_mail_body(wizard.move_id, wizard.mail_template_id, wizard.mail_lang)
-                wizard.mail_partner_ids = self._get_default_mail_partner_ids(wizard.move_id, wizard.mail_template_id, wizard.mail_lang)
-            else:
-                wizard.mail_subject = wizard.mail_body = wizard.mail_partner_ids = None
-
-    @api.depends('mail_template_id', 'sending_methods', 'extra_edis')
+            wizard.body = None
+
+            if wizard.template_id:
+                wizard.body = self._get_default_mail_body(wizard.move_id, wizard.template_id, wizard.lang)
+
+    @api.depends('template_id', 'sending_methods', 'extra_edis')
     def _compute_mail_attachments_widget(self):
         for wizard in self:
             manual_attachments_data = [x for x in wizard.mail_attachments_widget or [] if x.get('manual')]
             wizard.mail_attachments_widget = (
                 self._get_default_mail_attachments_widget(
                     wizard.move_id,
-                    wizard.mail_template_id,
+                    wizard.template_id,
                     extra_edis=wizard.extra_edis or {},
                     pdf_report=wizard.pdf_report_id,
                 )
                 + manual_attachments_data
             )
 
+    @api.depends('template_id')
+    def _compute_res_ids(self):
```

Comment:
We should probably add a comment on methods that are almost copy-pasted from `mail_compose_message`.
@tde-banana-odoo Hello :wave: Sorry in advance if it's not you that I should've pinged :slightly_smiling_face: 
I imagine your team is aware of this PR since LNA discussed on the task.
Is there anything we can do to help to ease the modification in case we happen to get some sort of embeddedable mail_compose_message ? If you have any suggestion they are more than welcome !
Getting the possibility to enable/disable the "Save template" behavior would be very cool too.